frontend/src/pages/_app.page.tsx

```diff
@@ -66,19 +65,11 @@ function MyApp({ Component, pageProps }: AppProps) {
       titleCase: false,
       debug: process.env.NEXT_PUBLIC_DEBUG === "true",
     });
-    const gaFields: ReactGa.FieldsObject = {
+    ReactGa.set({
       anonymizeIp: true,
       transport: "beacon",
-    };
-    const cookies = document.cookie.split("; ");
-    cookies.forEach((item) => {
-      if (item.trim().startsWith("utm_")) {
-        const cookieId = item.split("=")[0];
-        Object.assign(gaFields, convertUtmCookieToGaField(item));
-        clearCookie(cookieId);
-      }
     });
-    ReactGa.set(gaFields);
```

Comment:
note: this is the line that broke the GA reports for all the other traffic. maybe there's a way to fix just this line though ...

Reply:
See https://github.com/mozilla/fx-private-relay/pull/3775